[Service Offloading] Additional API Support to Kill Application

Chikdolman opened the pull request:
window.input_control.stopApplication("pkgName") is supported.
The API kills pkg of the Android application.
When terminating application from remote, stopApplication should be called.

Sunwoo-Nam commented:
Isn't there a build break in Windows build?

Chikdolman commented:
> Isn't there a build break in Windows build?

You are right. Fixed code for Windows build.

Sunwoo-Nam reviewed (approved):
LGTM.
It would be better to change the name of the module later.
ex) input control -> app control

Chikdolman committed:
[Service Offloading] Additional API Support to Kill Application

window.input_control.stopApplication("pkgName") is supported.
The API kills pkg of the Android application.
When terminating application from remote, stopApplication should be called.
